### PR TITLE
【translate】mds/heartbeat

### PR DESCRIPTION
**Pull request by SeanHai**
### What problem does this PR solve?

Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?

What's Changed:

How it Works:

Side effects(Breaking backward compatibility? Performance regression?): 

### Check List

- [ ] Relevant documentation/comments is changed or added
- [ ] I acknowledge that all my contributions will be made under the project's license

### CODE REVIEW

File: src/mds/heartbeat/chunkserver_healthy_checker.h [hunk 1]
```diff
@@ -48,25 +48,26 @@ struct HeartbeatOption {
...
+    // finds out that heartbeat is missed after heartbeatMissTimeOut peroid
     uint64_t heartbeatMissTimeOutMs;
 
-    // offLineTimeOutMs: 在offlineTimeOut的时间内如果没有收到chunkserver上报的心跳，//NOLINT
-    // 把chunkserver的状态置为offline, 并报警。
-    // schedule根据chunkserver的状态进行调度。
+    // offLineTimeOutMs:
+    // the maximun peroid that heartbeat is missed without
+    // setting the chunkserver to offline status and alarm.
+    // scheduling will depend on the status of chunkserver
```
Comment (xu-chaojie) [inline src/mds/heartbeat/chunkserver_healthy_checker.h:63]: on this status

File: src/mds/heartbeat/chunkserver_healthy_checker.h [hunk 2]
```diff
@@ -48,25 +48,26 @@ struct HeartbeatOption {
...
     uint64_t offLineTimeOutMs;
 
-    // cleanFollowerAfterMs： mds启动cleanFollowerAfterMs时间后
-    // 开启清理chunkserver上copyset的功能
+    // start cleaning copysets on chunkservers after
+    // running mds for this peroid of time
```
Comment (xu-chaojie) [inline src/mds/heartbeat/chunkserver_healthy_checker.h:67]: running  -> starting

File: src/mds/heartbeat/copyset_conf_generator.cpp [hunk 3]
```diff
@@ -36,8 +36,9 @@ bool CopysetConfGenerator::GenCopysetConf(
     const ::curve::mds::topology::CopySetInfo &reportCopySetInfo,
     const ::curve::mds::heartbeat::ConfigChangeInfo &configChInfo,
     ::curve::mds::heartbeat::CopySetConf *copysetConf) {
-    // topolgy不存在上报的copyset,
-    // 发一个空配置指导chunkserver将其删除
+    // reported copyset not exist on topology
```
Comment (xu-chaojie) [inline src/mds/heartbeat/copyset_conf_generator.cpp:39]: on topology  -> in topology

File: src/mds/heartbeat/heartbeat_manager.h [hunk 4]
```diff
@@ -77,105 +82,110 @@ class HeartbeatManager {
...
-     * @brief CheckRequest 检查心跳上报的request内容是否合法
+     * @brief CheckRequest Check the validity of a heartbeat request
      *
-     * @return 合法返回HeartbeatStatusCode::hbOK, 否则返回对应的错误码
+     * @return Return HeartbeatStatusCode::hbOK when valid, otherwise return
+     *         corresponding error code
      */
     HeartbeatStatusCode CheckRequest(const ChunkServerHeartbeatRequest &request);  // NOLINT
 
     // TODO(lixiaocui): 优化，统一heartbeat和topology中两个CopySetInfo的名字
```
Comment (xu-chaojie) [inline src/mds/heartbeat/heartbeat_manager.h:144]: 这个未翻译

File: src/mds/heartbeat/chunkserver_healthy_checker.h (hunk 1 above)
Comment (SeanHai) [reply src/mds/heartbeat/chunkserver_healthy_checker.h:63]: > on this status

done

File: src/mds/heartbeat/chunkserver_healthy_checker.h (hunk 2 above)
Comment (SeanHai) [reply src/mds/heartbeat/chunkserver_healthy_checker.h:67]: > running -> starting

done

File: src/mds/heartbeat/copyset_conf_generator.cpp (hunk 3 above)
Comment (SeanHai) [reply src/mds/heartbeat/copyset_conf_generator.cpp:39]: > on topology -> in topology

done

File: src/mds/heartbeat/heartbeat_manager.h (hunk 4 above)
Comment (SeanHai) [reply src/mds/heartbeat/heartbeat_manager.h:144]: > 这个未翻译

done

**Review by xu-chaojie — APPROVED** [review]

**Pull request merged**